Configure a blacklist of attributes that shouldn't be cloned (#23)
flagbit/akeneo-product-cloner: README.md

Put in a new **product code** and click on **save**. After that check if all the data is correct for the new product.

### Configuration

> You don't need to configure this bundle by default. It allows you to configure a blacklist of attribute codes though

Reviewer: ```suggestion
You don't need to configure this bundle by default.
```

> that won't be taken over to the cloned product or product model:

Reviewer: ```suggestion
The default behaviour is to clone all product attributes except the unique attributes.
In addition, you can specify a blacklist of attributes that shouldn't be cloned:
```

``` yaml
flagbit_product_cloner:
    attribute_blacklist:
        - your_attribute_code1
        - your_attribute_code2
        - your_attribute_code3
        ...
```

> Unique attributes won't be taken over to the clone by default.

Reviewer: ```suggestion
```
